src/travel/travel.js

```diff
@@ -46,27 +46,18 @@ function Travel({ slide, content, run, slideDone, executionId }) {
   // Rich text input sanitized
   const sanitizedtext = text ? parse(DOMPurify.sanitize(text, {})) : "";
 
-  // This is a shameful hack, hopefully temporarily, to combat the fact that the station-api
-  // only returns a station in _one direction_, instead of both
-  // After meeting*s* about this, we have concluded that the ids follow a pattern, where the last character
-  // in the string is defining which direction, and can be anything from 0-9.
-  // So, here we replace the last character of the id with 0-9, and then the rejseplan-api disregards ids that
-  // are not connected to a station (we hope).
   const getStationIds = () => {
     if (!(station instanceof Array) || station.length === 0) {
       return "";
     }
 
     let ids = "";
```

Comment:
Replace all below with:

```suggestion
      return station.reduce((carry, {id}) =>  `${carry}${id}@`, '');
```

The if (!station) is handled above...